enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/config/dto/SoapErrorResponseConfigDto.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.choreo.connect.enforcer.config.dto;
+
+/**
+ * Configuration for enabling SOAP error responses for SOAP APIs
```

Comment:
Let's add a period at the end. Applies to all other class and method comments. https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#format

Reply:
fixed in https://github.com/wso2/product-microgateway/pull/2972/commits/5c91048f9fa451e4b2f2d91dac704d3f6b755cd3

Reply:
Has to be fixed in all new java methods and classes.

Reply:
fixed in https://github.com/wso2/product-microgateway/pull/2972/commits/17a62e5eac8c71ad987bbbae2a42481ac4979fa6